app/app.go

```diff
@@ -467,6 +522,48 @@ func New(
 
 	planName := "v4.0.0"
 	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		// set the ICS27 consensus version so InitGenesis is not run
+		fromVM[icatypes.ModuleName] = icaModule.ConsensusVersion()
+
+		// create ICS27 Controller submodule params
+		controllerParams := icacontrollertypes.Params{
+			ControllerEnabled: false,
+		}
+
+		// create ICS27 Host submodule params
+		hostParams := icahosttypes.Params{
+			HostEnabled: false,
+			AllowMessages: []string{
+				"/cosmos.authz.v1beta1.MsgExec",
+				"/cosmos.authz.v1beta1.MsgGrant",
+				"/cosmos.authz.v1beta1.MsgRevoke",
```

Comment:
maybe remove authz and feegrant "/cosmos.staking.v1beta1.MsgCreateValidator", "/cosmos.vesting.v1beta1.MsgCreateVestingAccount", and "/cosmos.staking.v1beta1.MsgEditValidator".... 
they seem a bit esoteric for interchain accounts?

and maybe add Nft transfer? it could be potentially interesting e.g. if a Nft is owned by an interchain account that corresponds to a smart contract on Cronos?

Reply:
Done